libpius/signer.py

```diff
@@ -497,7 +497,7 @@ def import_clean_key(self, key):
     cmd = [self.gpg] + self.gpg_base_opts + self.gpg_quiet_opts + [
         '--no-default-keyring',
         '--keyring', self.tmp_keyring,
-        '--import-options', 'import-minimal',
+        '--import-options', 'import-minimal,keep-ownertrust',
```

Comment:
Yeah, I like that, actually, that's nice and clean. Sorry for the delay, it's been a crazy month of travel and work and such. :)